makerdao/governance-portal-v2
# Daijs ethers update

### Link to Clubhouse story
https://app.shortcut.com/dux-makerdao/story/392/update-web3-js-ethers-js-in-the-plugin-for-eip-1559

### What does this PR do?
Brings in the dai.js plugins which have removed web3.js and use ethers.js throughout

### Steps for testing:
Things we need to test (feel free to add)
1. making transactions and monitoring return values
2. make sure switching accounts is done correctly and txns are sent from the correct account
3. switching networks
4. run/fix tests

### Screenshots (if relevant):

### Any additional helpful information?:

### Add a GIF:
